docs/guides/networking/linode-network/ip-failover-bgp-frr/index.md

```diff
@@ -0,0 +1,300 @@
+#
+bgpd=yes
+{{</ file >}}
+
+1.  Restart the FRR service:
+
+        sudo systemctl restart frr.service
+
+## Configure FRR
+
+With FRR installed, you can now configure it to enable IP failover.
+
+1.  Gather the following information, which is required for the next step:
+
+    - **Shared IP address** (`[SHARED_IP]`): The shared IP address you've configured for both the primary and secondary instances. See [Configure IP Sharing](#configure-ip-sharing).
+    - **Hostname** (`[HOSTNAME]`): The hostname defined on the Compute Instance you are configuring (ex: `atl-bgp-1.example.com`).
+    - **Role** (`[ROLE]`): The role of this Compute Instance within your failover strategy.
+      - `primary`: All requests are routed to this Compute Instance, provided it is accessible.
+      - `secondary`: If the `primary` instance fails, all requests are routed to this Compute Instance, provided it is accessible.
+    - **Data center ID** (`[DC_ID]`): The ID of this data center as defined by the list below:
+        - Atlanta (USA): `4`
+        - Dallas (USA): `2`
+        - Frankfurt (Germany): `10`
+        - Fremont (USA): `3`
+        - London (UK): `7`
+        - Mumbai (India): `14`
+        - Newark (USA): `6`
+        - Singapore: `9`
+        - Sydney (Australia): `16`
+        - Tokyo (Japan): `11`
+        - Toronto (Canada): `15`
+
+1.  The template below should be used for the FRR configuration on Compute Instance within your IP failover setup. Ensure you replace any instances of `[SHARED_IP]`, `[HOSTNAME]`, `[ROLE]`, and `[DC_ID]` as outlined above. Store the template somewhere that you can easily access later. In a later step, you copy the contents of the template and paste them into the VTY interactive shell.
+
+      {{< file "~/ipfailover.conf">}}
+hostname [HOSTNAME]
+
+router bgp 65000
+no bgp ebgp-requires-policy
+coalesce-time 1000
+bgp bestpath as-path multipath-relax
+neighbor RS peer-group
+neighbor RS remote-as external
+neighbor RS capability extended-nexthop
+neighbor 2600:3c0f:[DC_ID]:34::1 peer-group RS
+neighbor 2600:3c0f:[DC_ID]:34::2 peer-group RS
+neighbor 2600:3c0f:[DC_ID]:34::3 peer-group RS
+neighbor 2600:3c0f:[DC_ID]:34::4 peer-group RS
+address-family ipv4 unicast
+  network [SHARED_IP]/32 route-map [ROLE]
+  redistribute static
+exit-address-family
+route-map primary permit 10
+set large-community 63949:1:1
+route-map secondary permit 10
+set large-community 63949:1:2
+{{</ file >}}
+
+1.  Run the VTY shell:
+
+        sudo vtysh
+
+1.  Enter configuration mode:
+
+        conf t
+
+1.  Copy the contents of your template configuration file and paste them into the VTY shell:
+
+1.  Tell the VTY shell that you are done entering your configurations:
+
+        end
+
+1. Write your configurations to VTY:
+
+        write
+
+1.  Verify that the configurations you entered were correctly written by showing VTY's running configuration:
+
+        show running-config
+
+1.  Exit out of the VTY shell:
+
+        q
```

Comment:
I'd replace all these steps with simply pasting and editing that file in `/etc/frr/frr.conf` and `systemctl restart frr` afterwards, its easier than dealing with vtysh